src/main/resources/META-INF/resources/pages/map.html

```diff
@@ -15,6 +15,7 @@
   <footer>
     <script src="/js/widgets/toastWindow.js"></script>
     <script src="/js/widgets/missionLog.js"></script>
+    <script src="/js/widgets/gameMenu.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
```

Comment:
Since the game menu uses jquery, this should get included earlier.  Should probably move it to first in the script includes.